### PR TITLE
Add `4.x` branch for doctrine mongodb admin

### PR DESCRIPTION
**Pull request by franmomu**

### CODE REVIEW

File: config/projects.yaml [hunk 1]
```diff
@@ -229,7 +229,14 @@ doctrine-mongodb-admin-bundle:
             php_extensions: ['mongodb-1.9.0', 'zip']
             target_php: '7.4'
             variants:
-                symfony/symfony: ['4.4.*']
+                symfony/symfony: ['4.4.*', '5.3.*']
+                sonata-project/admin-bundle: ['^4@rc']
+        4.x:
+            php: ['7.3', '7.4', '8.0']
+            php_extensions: ['mongodb-1.9.0', 'zip']
+            target_php: '7.4'
```
Comment (jordisala1991) [inline config/projects.yaml:237]: why do you need to restrict this php version?

Comment (franmomu) [reply config/projects.yaml]: I don't remember if it was because some static analysis issue, I've remove them.

Comment (jordisala1991) [reply config/projects.yaml]: Maybe you want to check first on the repo with a sample Pr

Comment (franmomu) [reply config/projects.yaml]: Apparently they are fine:

https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/633
https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/pull/634

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
You need to rebase

**Pull request closed without merging**